scripts/uberenv/uberenv.py

```diff
@@ -322,6 +322,10 @@ def setup_paths_and_dirs(self):
         self.pkgs = pjoin(self.uberenv_path, "packages","*")
 
         # setup destination paths
+        if self.opts["prefix"] is None:
```

Comment:
This will probably end up getting overwritten next time we update uberenv.  We will have to stand guard against it and be vigilant...

Reply:
Should I add a comment like "SERAC EDIT"?

Reply:
We could make a patch of it and add it to the "scripts/gitlab/update-uberenv.sh" script.

(note: I like commenting merged PRs ;))